OpenRA.Game/Map/CellLayer.cs

```diff
@@ -155,7 +155,7 @@ public MPos Clamp(MPos uv)
 	// Helper functions
 	public static class CellLayer
 	{
-		/// <summary>Create a new layer by resizing another layer.  New cells are filled with defaultValue.</summary>
```

Comment:
this looks like html to me, and multiple whitespace characters get filtered on display? or are tags like these used by IDEs too?